sqlmesh/core/renderer.py

```diff
@@ -90,12 +90,22 @@ def _render(
             The rendered expressions.
         """
 
-        cache_key = self._cache_key(start, end, execution_time, runtime_stage)
-        start_dt, end_dt, execution_dt, runtime_stage = cache_key
-        if cache_key not in self._cache:
+        cache_key = self._cache_key(start, end, execution_time)
+        start_dt, end_dt, execution_dt = cache_key
+        if runtime_stage != RuntimeStage.LOADING or cache_key not in self._cache:
```

Comment:
Was this a bug?

Reply:
Do we ignore cache when not loading?

Reply:
it's kind of unnecessary, the cache is really to prevent recomputation for loads, and so i removed the complexity of runtime stage in cache, 

it just means that when evaluating / creating tables, we won't hit the query cache, but that's fine, and may be intentional if your sql has side effects

Reply:
going to revert this

Reply:
Did it end up causing issues?

Reply:
it's annoying because i need to also not set the cache when the runtime stage is not loading